AsyncDisplayKit/ASMapNode.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <AsyncDisplayKit/AsyncDisplayKit.h>
+#import <MapKit/MapKit.h>
+
+@interface ASMapNode : ASImageNode
+
+- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate NS_DESIGNATED_INITIALIZER;
```

Comment:
Aha, doesn't this need to be an MKMapRegion or something else that fully specifies the visible extent?

Later - not necessary for you to do personally @aaronschubert0, more like feature request - we could add amazing helper functions that aid a user who has a CLLocationCoordinate2D and perhaps a radius as an expression of zoom scale, and we could create a region for them.

I think I will merge this diff now, but only because 2.0 is coming up and I feel confident to make a breaking API change in that release.  We really should change this though because right now the zoom level seems unspecified, and it would force creation of the mapView if set through there.